mne/io/eeglab/eeglab.py

```diff
@@ -0,0 +1,374 @@
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+import warnings
+
+from ...utils import logger, verbose
+from ..utils import _read_segments_file
+from ..meas_info import _empty_info, create_info
+from ..base import _BaseRaw, _check_update_montage
+from ..constants import FIFF
+from ...channels.montage import Montage
+from ...epochs import _BaseEpochs
+from ...event import read_events
+from ...externals.six import string_types
+
+
+def _get_info(eeg, montage, eog=None):
+    """Get measurement info.
+    """
+    info = _empty_info(sfreq=eeg.srate)
+    info['nchan'] = eeg.nbchan
+
+    # add the ch_names and info['chs'][idx]['loc']
+    path = None
+    if len(eeg.chanlocs) > 0:
+        ch_names, pos = [], []
+        kind = 'user_defined'
+        selection = np.arange(len(eeg.chanlocs))
+        locs_available = True
+        for chanloc in eeg.chanlocs:
+            ch_names.append(chanloc.labels)
+            loc_x = chanloc.X if len(chanloc.X) > 0 else 0.
+            loc_y = chanloc.Y if len(chanloc.Y) > 0 else 0.
+            loc_z = chanloc.Z if len(chanloc.Z) > 0 else 0.
+            locs = np.r_[loc_x, loc_y, loc_z]
+            if np.unique(locs).size == 1:
+                locs_available = False
+                break
+            pos.append(locs)
+        if locs_available:
+            montage = Montage(np.array(pos), ch_names, kind, selection)
+    elif isinstance(montage, string_types):
+        path = op.dirname(montage)
+
+    if montage is None:
+        info = create_info(ch_names, eeg.srate)
+    else:
+        _check_update_montage(info, montage, path=path,
+                              update_ch_names=True)
+
+    info['buffer_size_sec'] = 1.  # reasonable default
+    # update the info dict
+    cal = 1e-6
+    if eog is None:
+        eog = []
+    for ch in info['chs']:
+        ch['cal'] = cal
+        if ch['ch_name'].startswith('EOG') or ch['ch_name'] in eog:
```

Comment:
Yes, I really don't like this. I never designate EOG channels because I don't see any reason for treating them differently, and the EEGLAB people for example have written whole papers about treating them like regular channels. I think having to rename channels in that situation would be very inelegant.

Also, typical EOG channel names are `'VEOG'` or `'HEOG'`, which won't be caught by this either.